packages/ses/test/test-console-error-trap.js

```diff
@@ -0,0 +1,24 @@
+import test from 'ava';
+import { exec } from 'child_process';
+import { dirname, join } from 'path';
+
+const cwd = join(
+  dirname(new URL(import.meta.url).pathname),
+  'console-error-trap',
+);
+
+test.cb('errors reveal their stacks', t => {
+  t.plan(3);
+  exec('node index.js', { cwd }, (err, stdout, stderr) => {
+    t.assert(err, 'exit code should be non-zero');
+    t.assert(
+      stderr.includes('(Error#1)'),
+      'stderr should have an error marker',
+    );
+    t.assert(
+      stdout.includes('Error#1: Shibboleth'),
+      'stdout should contain error message',
+    );
```

Comment:
Why is some console output going to stdout and some going to stderr? Did I screw something up in the console implementation?

Reply:
Aye, we should make sure to use `console.log` only for `stdout`. We currently use a mix.

Reply:
https://github.com/endojs/endo/issues/805